commit cfa662f8745a92bdd820be5669d13cf621bbf7e3
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-09-05T08:58:03-07:00 (5 weeks ago)
    
    TRITON-1840 expose provisioning limits through CloudAPI
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/docs/index.md b/docs/index.md
index 5d7b7a91..dde36cc9 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -1273,7 +1273,7 @@ An array of provisioning limit objects. Possible values for each provisioning
 limit object are:
 
 **Field**   | **Type** | **Description**         | **Possible values**
------------ | -------- | ---------------
+----------- | -------- | ----------------------- | -------------------
 by          | String   | The type of limit. Note that "machines" means the "number of machines". | "ram", "quota", or "machines".
 value       | Number   | The limit value. I.e. the total number of machines, total ram or total quota. | Ram is defined in terms of MiB. Quota is defined in terms of GiB.
 used        | Number   | How much of the limit is used by existing machines. | Ram is defined in terms of MiB. Quota is defined in terms of GiB.
